network-area-diagram/src/main/java/com/powsybl/nad/build/iidm/VoltageLevelFilter.java

```diff
@@ -62,17 +62,31 @@ public static VoltageLevelFilter createNominalVoltageLowerBoundFilter(Network ne
         return createNominalVoltageFilter(network, voltageLevelIds, nominalVoltageLowerBound, Double.MAX_VALUE, depth);
     }
 
+    public static VoltageLevelFilter createNominalVoltageLowerBoundFilter(Network network, double nominalVoltageLowerBound) {
+        return createNominalVoltageFilter(network, nominalVoltageLowerBound, Double.MAX_VALUE);
+    }
+
     public static VoltageLevelFilter createNominalVoltageUpperBoundFilter(Network network, List<String> voltageLevelIds, double nominalVoltageUpperBound, int depth) {
         return createNominalVoltageFilter(network, voltageLevelIds, 0, nominalVoltageUpperBound, depth);
     }
 
+    public static VoltageLevelFilter createNominalVoltageUpperBoundFilter(Network network, double nominalVoltageUpperBound) {
+        return createNominalVoltageFilter(network, 0, nominalVoltageUpperBound);
+    }
+
     public static VoltageLevelFilter createNominalVoltageFilter(Network network, List<String> voltageLevelIds,
                                                                 double nominalVoltageLowerBound, double nominalVoltageUpperBound,
                                                                 int depth) {
-        checkVoltageBoundValues(nominalVoltageLowerBound, nominalVoltageUpperBound);
-        Predicate<VoltageLevel> voltageLevelPredicate = voltageLevel -> voltageLevel.getNominalV() >= nominalVoltageLowerBound && voltageLevel.getNominalV() <= nominalVoltageUpperBound;
+        return createVoltageLevelFilterWithPredicate(network, voltageLevelIds, depth, getPredicateFromVoltageBounds(nominalVoltageLowerBound, nominalVoltageUpperBound));
+    }
 
-        return createVoltageLevelFilterWithPredicate(network, voltageLevelIds, depth, voltageLevelPredicate);
+    public static VoltageLevelFilter createNominalVoltageFilter(Network network, double nominalVoltageLowerBound, double nominalVoltageUpperBound) {
+        return createNominalVoltageFilterWithPredicate(network, getPredicateFromVoltageBounds(nominalVoltageLowerBound, nominalVoltageUpperBound));
+    }
+
+    private static Predicate<VoltageLevel> getPredicateFromVoltageBounds(double nominalVoltageLowerBound, double nominalVoltageUpperBound) {
```

Comment:
`getPredicateFromNominalVoltageBounds` would be a better name (since the filter consider the nominal voltage).

Reply:
Indeed, it is clearer!